### DISCUSSION TITLE
Aqara FP1E - what is the latest Firmware version available?

### DISCUSSION THREAD

**Discussion by Aaroneisele55** [Koenkk/zigbee2mqtt#25211] [Devices]
Hello,
I have recently set up three of Aqara's new FP1E sensors, and whike they work great and consistently detect presence, even while sleeping, I wondered if I would be able to update the firmware once Aqara releases new versions...
I am currently on version 26, but I don't have an Aqara hub to check or update the FW...
Where could I get information about new versions?

Best regards
Aaron

**Comment by Nick425321** [comment]
I have a M2 hub and I'm just updating a FP1E from 026 to 028, the release notes are very chat gpt'ish, basically all it says is they improved it's performance.

**Reply by Aaroneisele55** [reply]
> I have a M2 hub and I'm just updating a FP1E from 026 to 028, the release notes are very chat gpt'ish, basically all it says is they improved it's performance.

Thanks for the info, did you notice any improvements after the update?

**Reply by Nick425321** [reply]
It is to early to say, I have not had them connected to HA for several weeks as they are sometimes extremely difficult to pair, some how I did manage to pair 4 more last night, I'm just going leave them connected for a week or two before using them in automations again.

**Reply by Nick425321** [reply]
This morning 3 of 5 were unavailable so changing firmware changed nothing for me. I will be returning them while I still can, as they have the same issue as the FP1, they become unavailable and almost impossible to rejoin.

**Comment by jorisvervuurt** [comment]
The device you mentioned seems to support OTA updates via Zigbee2MQTT: https://www.zigbee2mqtt.io/devices/FP1E.html

Open the Zigbee2MQTT web interface and click the OTA tab. Then click 'Check all' in the top right corner.

**Reply by Aaroneisele55** [reply]
The OTA `check all` button didn't bring any result, still no update available....

I just checked https://github.com/Koenkk/zigbee-OTA/tree/master/images/Lumi and there seems to be no image available for the model `lumi.sensor_occupy.agl1` that I have... 
Could it be the case that while the hardware and software would technically support OTA, Aqara ("Lumi") doesn't provide image files?
